boxoffice/models/line_item.py

```diff
@@ -189,6 +189,20 @@ def fetch_all_details(cls, item_collection):
         return db.session.execute(line_item_query).fetchall()
 
     @classmethod
+    def fetch_assignee_details(cls, item_collection):
+        """
+        Returns invoice_no, ticket title, assignee fullname, assignee email, assignee phone and assignee details
+        for all the line items in a given item collection
+        """
+        line_item_join = db.outerjoin(cls, Assignee, db.and_(LineItem.id == Assignee.line_item_id,
```

Comment:
This should be a `join` and not an `outerjoin`. `join` will include all `line items` regardless of whether they have been assigned.